.circleci/config.yml

```diff
@@ -292,3 +300,51 @@ jobs:
             - target/debug/build
             - target/debug/deps
           key: cargocache-v2-lint-rust:1.44.1-{{ checksum "Cargo.lock" }}
+
+  # This job roughly follows the instructions from https://circleci.com/blog/publishing-to-github-releases-via-circleci/
+  build_and_upload_contracts:
+    docker:
+      # Image from https://github.com/cibuilds/github, based on alpine
+      - image: cibuilds/github:0.13
+    steps:
+      - run:
+          name: Install Docker client
+          command: apk add docker-cli
+      - setup_remote_docker
+      - checkout
+      - run:
+          # We cannot mount local folders, see https://circleci.com/docs/2.0/building-docker-images/#mounting-folders
+          name: Prepare volume with source code
+          command: |
+            # create a dummy container which will hold a volume with config
+            docker create -v /code --name with_code alpine /bin/true
+            # copy a config file into this volume
+            docker cp Cargo.toml with_code:/code
+            docker cp Cargo.lock with_code:/code
+            docker cp ./contracts with_code:/code
+            docker cp ./packages with_code:/code
+      - run:
+          name: Build development contracts
+          command: |
+            docker run --volumes-from with_code cosmwasm/rust-optimizer:0.10.0
+            docker cp with_code:/code/artifacts ./artifacts
+            mv artifacts/contracts.txt artifacts/checksums.txt
```

Comment:
If we do that, we better call the file `checksums.txt` everywhere. I used that name in ComsJS but we can switch it to `contracts.txt`. However, I think `checksums.txt` is a bit more user friendly.

Reply:
Okay, I will update rust-optimizer to 0.10.1 with that naming. Then I can merge this, right?

Reply:
Yeah, looks good to me. Just the deactivated publishing is missing but this you will figure out by bruteforceing it

Reply:
Addressed by https://github.com/CosmWasm/rust-optimizer/pull/20

Reply:
Perfect. We'll benefit from this consistency in the long run